doc/Extensions/Authentication.md

```diff
@@ -182,17 +182,20 @@ An example config setup for use with Jumpcloud LDAP as a service is:
 
 ```php
 $config['auth_mechanism'] = "ldap";
+unset($config['auth_ldap_group']); #Remove built-in group
```

Comment:
This can be omitted it is unneeded.  LibreNMS removes the default group if it is unchanged.

Reply:
With this line removed the setting still remains in the "Global Settings" page as:
`auth_ldap_group | cn=groupname,ou=groups,dc=example,dc=com`

I would rather not see the default or placeholder setting.  Though, it seems like this parameter isn't required at all for successful authentication.

Reply:
@dsmfool yeah, the code discards any entries that are set to that.

Feel free to keep the unset in your local config.  I just don't think it should be in the docs ;)

Reply:
I agree that it shouldn't have to be unset in the docs, but it should perhaps be shown how to remove them or overwrite them until the default groups are removed.

I'm not very familiar with Git, and unaware how to modify my commit for this pull request to reflect these requested changes to my pull request.

Reply:
@dsmfool go to the files changed tab and click the edit button.  That will allow you to edit this file.

You can cancel and browse around to other files in that branch to edit them too if you want.

Reply:
Done.  Hopefully I did it correctly.